### PR TITLE
Update isilon_create_directories.sh

### PR DESCRIPTION
**Pull request by bonibruno**
Added the creation of anonymous user directory as it is needed by Hive when using Isilon for HDFS Tiered Storage with a DAS Cluster.

### CODE REVIEW

File: isilon_create_directories.sh [hunk 1]
```diff
@@ -202,6 +202,7 @@ case "$DIST" in
             "/user/hive#700#hive#hdfs" \
             "/user/hue#755#hue#hue" \
             "/user/oozie#775#oozie#hdfs" \
+	    "/user/anonymous#775#hdfs#hdfs" \
```
Comment (dmtucker) [inline isilon_create_directories.sh]: It best to avoid mixing tabs and spaces.

**Comment by bonibruno** [comment]
As stated, this is required when using Isilon for HDFS Tiering with a DAS cluster.

This applies to any OneFS release.

File: isilon_create_directories.sh (hunk 1 above)
Comment (bonibruno) [reply isilon_create_directories.sh]: Sure, when hive.server2.authentication is set to none (default), and no user name is specified in the client connection to hive, driver will default to using anonymous as the user name.  When using beeline, for example, to connect to Isilon (or any DAS cluster) as a remote HDFS filesystem (one of our HTS use cases), the driver checks if /user/anonymous exists on the remote filesystem exists and permissions, if the remote /user/anonymous directory doesn’t exist, you immediately get “/user/anonymous directory missing error”  and beeline fails to connect.  If permissions are wrong, then you get a permissions denied error.  Adding /user/anonymous to Isilon with the right permissions fixes the problem and anonymous connections from beeline work fine thereafter.

**Comment by bonibruno** [comment]
I've updated field documentation to specify a user with Hive CLI or Beeline client, this avoids the need to create anonymous user with our scripts.

**Pull request closed without merging**